src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/NativePal.Linux.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.DirectoryServices.Protocols
+{
```

Comment:
I am not sure if it is worth moving al interop things under its own folder and namespace?